### PR TITLE
feat(js/plugins/checks): checks evaluator plugin returns multiple scores

### PR DESCRIPTION
**Pull request by HunterHeston**
Update checks evaluators to return multiple results, one per requested policy. 

Checklist (if applicable):
- [x] Tested (manually, unit tested, etc.)
- [ ] Changelog updated
- [ ] Docs updated

### CODE REVIEW

File: js/plugins/checks/src/evaluation.ts [hunk 1]
```diff
@@ -134,15 +140,20 @@ function createPolicyEvaluator(
...
-        },
-        testCaseId: datapoint.testCaseId,
-      };
+        }
+      });
+
+      return {
+        evaluation: evaluationResults,
+        testCaseId: datapoint.testCaseId
+      }
```
Comment (HunterHeston) [inline js/plugins/checks/src/evaluation.ts:156]: @ssbushi Please take a look.

**Comment by pavelgj** [comment]
please follow https://www.conventionalcommits.org/en/v1.0.0/ for PRs

File: js/plugins/checks/src/evaluation.ts [hunk 2]
```diff
@@ -104,15 +116,14 @@ function createPolicyEvaluator(
   projectId: string,
   auth: GoogleAuth,
   ai: Genkit,
-  policy_config: ChecksEvaluationMetricConfig
+  policy_config: ChecksEvaluationMetricConfig[],
```
Comment (ssbushi) [inline js/plugins/checks/src/evaluation.ts:119]: This still looks like it is defining one evaluator per policy (which will make one API call per policy). 

You'd want to define a single genkit evaluator that accepts an array of policies and evaluates for all of them. The response can include multiple `ScoreSchema` objects.

**Review by ssbushi — CHANGES_REQUESTED** [review]
See above comment.

File: js/plugins/checks/src/evaluation.ts (hunk 2 above)
Comment (HunterHeston) [reply js/plugins/checks/src/evaluation.ts:119]: I left the original logic of creating an evaluator instance for each policy and an evaluator for all the policies configured. The thought was that some users might want to just run one of the policies individually.

That happens here in the constructor:
```ts
  // Individual evaluators, one per configured metric.
  const evaluators = policy_configs.map((policy_config) => {
    return createPolicyEvaluator(
      projectId,
      auth,
      ai,
      [policy_config],
      policy_config.type as string
    );
  });

  // Single evaluator instnace with all configured policies.
  evaluators.push(
    createPolicyEvaluator(projectId, auth, ai, policy_configs, 'all_metrics')
  );
```ts
I'm not sure if that's the right approach though. I'm leaning towards just removing all the individual evaluators and having a single evaluator named checks that runs all configured policies every time. If a user wants to run a different set of policies or one in isolation then they would have to update their plugin config. What do you think?

Comment (HunterHeston) [reply js/plugins/checks/src/evaluation.ts]: @ssbushi removed the extra evaluators. Users can now just run `genkit eval:run test-dataset.json --evaluators=checks`

File: js/plugins/checks/src/evaluation.ts [hunk 3]
```diff
@@ -77,11 +77,7 @@ export function checksEvaluators(
     }
   );
 
-  const evaluators = policy_configs.map((policy_config) => {
-    return createPolicyEvaluator(projectId, auth, ai, policy_config);
-  });
-
-  return evaluators;
+  return [createPolicyEvaluator(projectId, auth, ai, policy_configs)];
```
Comment (ssbushi) [inline js/plugins/checks/src/evaluation.ts:80]: Why not just make this return an `EvaluatorAction` instead of an array?

**Review by ssbushi — APPROVED** [review]

File: js/plugins/checks/README.md [hunk 4]
```diff
@@ -78,16 +78,11 @@ Create a JSON file with the data you want to test. Add as many test cases as you
...
 ```bash
-# Run just the DANGEROUS_CONTENT classifier.
-genkit eval:run test-dataset.json --evaluators=checks/dangerous_content
-```
-
-```bash
-# Run all classifiers.
-genkit eval:run test-dataset.json --evaluators=checks/dangerous_content,checks/pii_soliciting_reciting,checks/harassment,checks/sexually_explicit,checks/hate_speech,checks/medical_info,checks/violence_and_gore,checks/obscenity_and_profanity
+# Run all configured classifiers.
+genkit eval:run test-dataset.json --evaluators=checks
```
[the remark this answers is not in the document]
Comment (HunterHeston) [reply js/plugins/checks/README.md]: Updated!

File: js/plugins/checks/src/evaluation.ts (hunk 3 above)
Comment (HunterHeston) [reply js/plugins/checks/src/evaluation.ts]: done!

**Pull request merged**